newrelic.js

```diff
@@ -5,15 +5,8 @@
  * See lib/config/default.js in the agent distribution for a more complete
  * description of configuration variables and their potential values.
  */
+
 exports.config = {
```

Comment:
I'm not against removing the app_name and license_key from the config. I noticed the newrelic npm package didn't need these vars to be explicitly stated in the config if they were declared in the .env. 

Just want to note one thing: The original code worked for the server side monitoring (I think). What's nice about the original implementation is that we only needed to declare the license key in the .env - which means that we don't have NEW_RELIC_APP_NAME declared in any environment (vercel, qa, or prod) atm. This means that with this change, we will need to add the NEW_RELIC_APP_NAME var to the vercel .env, the task definitions for qa and prod, and the CFTs for qa and prod. _Otherwise newrelic will not work._

Reply:
Right. I don't think there's a way around this. I tried the original version where app name and the license are set in this config and it does _not_ work. It may be the new (updated) NR npm package. So this is now the way it is configured and we _have_ to set the env var in all server environments.

Reply:
gotcha. that's fine - just wanted to flag the new env var so we don't forget that it's a requirement for this to work.